dali/python/nvidia/dali/_utils/eager_utils.py

```diff
@@ -604,8 +604,11 @@ def _choose_device(op_name, wrapper_name, inputs, device_param):
         device = device_param
         device_id = 0
 
-    if device == "cpu" and input_device == "gpu":
-        raise ValueError("An operator with device='cpu' cannot accept GPU inputs.")
+    # TODO(michalz): Verify against InputDevice from the schema.
```

Comment:
So now in the eager mode we do the copy as well?

Reply:
It's not about doing the copy but even allowing a CPU operator to take a GPU input.